src/applications/appeals/testing/sc/pages/facilityTypes.js

```diff
@@ -0,0 +1,42 @@
+import {
+  textUI,
+  textSchema,
+  checkboxGroupUI,
+  checkboxGroupSchema,
+} from 'platform/forms-system/src/js/web-component-patterns';
+
+import {
+  facilityTypeTitle,
+  facilityTypeTextLabel,
+  facilityTypeChoices,
+} from '../content/facilityTypes';
+
+const labels = Object.keys(facilityTypeChoices);
+
+export default {
+  uiSchema: {
+    facilityTypes: {
+      ...checkboxGroupUI({
```

Comment:
Optional suggestion, we could also consider adding an additional pattern like checkboxGroupWithTextUI to make it easier to consume this pattern

Reply:
That sounds great, but let's consider this change for a separate PR.